pkgs/by-name/op/openscad-unstable/package.nix

```diff
@@ -95,6 +96,7 @@ clangStdenv.mkDerivation rec {
     python3
     libsForQt5.qt5.wrapQtAppsHook
     llvmPackages.bintools
+    wrapGAppsHook
```

Comment:
Aren't the binaries now double wrapped?

Reply:
Good question. Tbh, I don't know.
`wrapQtAppsHook` seems to be required for packaging Qt apps, https://nixos.wiki/wiki/Qt#Packaging.
`wrapGAppsHook` is only for `GLib`/`GSettings`, https://nixos.org/manual/nixpkgs/stable/#ssec-gnome-hooks

I do not know how the hooks actually works or affects the different `preFix`- phases or `fixupOutputHooks`. Maybe you can help me figure that out?

Reply:
See https://github.com/NixOS/nixpkgs/blob/0cfd293af3249b214085553e0ef053141f32099a/doc/languages-frameworks/gnome.section.md#when-using-wrapgappshook-with-special-derivers-you-can-end-up-with-double-wrapped-binaries-ssec-gnome-common-issues-double-wrapped on how to do that

Reply:
Ok, thanks for the link. That's nice of you.
I will look into it during the weekend.